scss/crm-dashboard.scss

```diff
@@ -0,0 +1,31 @@
+@import 'SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/variables';
+@import 'SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/vendor/font-awesome/mixins';
+
+.___outside-namespace {
```

Comment:
Lets not put all these in a single file. Lets create two folders
1. `civicase`
2. `crm-dashboard`

Inside civicase folder move all existing folders of scss(except `base`)
And inside `crm-dashboard` we need to create a folder called `page` and `component` and move the following scss accordingly.

Reply:
As discussed in chat, moved all the files inside the Civicase project. Decided to make a tabs component `civicase__tabs` and this particular one is dark by implementing the `civicase__tabs--dark` class.